dashboard/src/pages/ChapterMembers.vue

```
@@ -112,10 +112,10 @@ import ChapterHeader from '@/components/ChapterHeader.vue'
 const session = inject('$session')
 const route = useRoute()
 
-const isLead = () => {
+const isCoreTeam = () => {
     let currentUser = chapter.doc.chapter_members.filter((m) => m.email === session.user)
```

ESLint

[error] Delete `··`

(prettier/prettier)